doc/source/parameters/cfd/cfd.rst

```diff
@@ -0,0 +1,172 @@
+***********************************************
+Computational Fluid Dynamics and Multiphysics
+***********************************************
+
+
+Simulation Control 
+------------------
+
+This subsection contains the general information of the simulation, including the time integration method and the general output names. It is the most commonly modified section for a simulation. A standard convention in Lethe is to keep this section at the top of the parameter file, since it is generally the most commonly accessed one.
```

Comment:
```suggestion
This subsection contains the general information of the simulation, including the time integration method and the general output names. It is the most commonly modified section for a simulation. A standard convention in Lethe is to keep this section at the top of the parameter file, since it is generally the most accessed one.
```